rel/files/tracer_connect.js

```diff
@@ -0,0 +1,5 @@
+
+function open_websocket() {
```

Comment:
Yes, but it is supposed to end up in the other place because it is there. We need to decide how the build and deployment process should look like and what goes where.